codespell_lib/data/dictionary.txt

```diff
@@ -18282,7 +18284,9 @@ nighboring->neighboring
 nightfa;;->nightfall
 nightime->nighttime
 nimutes->minutes
+nin->min
 nineth->ninth
+ninimum->minimum
 ninjs->ninja
 ninteenth->nineteenth
 ninties->1990s
```

Comment:
Oh wow, this seems like a bit of an extreme "correction"! Should probably just be ninties->nineties ?
You wouldn't really want "My grandma lived into her ninties" to be corrected to "My grandma lived into her 1990s" :rofl:

Reply:
Good spot, do you want to open a new PR for them given they're entirely unrelated to this one and it's growing rather large already? I suspect they should be in informal probably. There are probably 80s/70s etc variants too.

Reply:
> Should probably just be ninties->nineties ?

I've added that option as the primary choice.

> EDIT: Also, perhaps the `ninty->ninety` just below here should be `ninty->ninety, minty,`?

Done this.

Reply:
> There are probably 80s/70s etc variants too.

Nope :slightly_smiling_face: 

```bash
$ grep '[0-9]' codespell_lib/data/dictionary.txt 
1nd->1st
2rd->2nd
2st->2nd
3nd->3rd
3st->3rd
nd->and, 2nd,
ninties->1990s
p0enis->penis
sh1sum->sha1sum
UTF8ness->UTF-8-ness
```